src/AccessibilityInsights/Modes/SnapshotModeControl.xaml.cs

```diff
@@ -393,11 +393,11 @@ public void CopyToClipboard()
                 var se = this.ctrlHierarchy.SelectedElement ?? this.ElementContext.Element;
 
                 // glimpse
-                sb.AppendFormat(CultureInfo.InvariantCulture, "Glimpse: {0}", se.Glimpse);
+                sb.AppendFormat(CultureInfo.InvariantCulture, Properties.Resources.SnapshotModeControl_Glimpse + ": {0}", se.Glimpse);
```

Comment:
The {0} should be inside the localized string. Also, we should really use `CultureInfo.CurrentCulture` when formatting with a string resource. I don't think it would make a difference here, but it's a good practice. I imagine that we violate this a lot, but change has to start somewhere